bin/ensure-up-to-date

```diff
@@ -26,7 +26,9 @@ sleep 5
 ./bin/openapi3/php-petstore.sh
 
 # Check:
+echo "Perform git diff"
```

Comment:
I would have let this 3 lines (`echo`, `git diff` and `git status`) inside the if... We do not need them in the OK case.